src/Microsoft.SqlTools.ServiceLayer/Utility/ServiceLayerCommandOptions.cs

```diff
@@ -27,7 +27,7 @@ public ServiceLayerCommandOptions(string[] args) : base(args.Where(arg => !servi
         {
             for (int i = 0; i < args.Length; ++i)
             {
-                string arg = args[i].ToLowerInvariant();
+                string arg = args[i]?.ToLowerInvariant();
```

Comment:
I assume args is coming from Main() and are there for "guaranteed" not to be null.  But the check should hurt anything.